app/lib/screens/patient_screen.dart

```diff
@@ -20,42 +24,243 @@ class PatientScreen extends StatefulWidget {
+    return SingleChildScrollView(
+        physics: const AlwaysScrollableScrollPhysics(),
+        child: Column(children: [
+          Card(
+              child: PatientOverview(
+                  initiallyExpanded: expanded, patient: patient)),
+          ...children
+        ]));
+  }
+
+  Widget _buildPerformedAction(PerformedAction performedAction) {
+    return Card(
+        child: Padding(
+            padding: const EdgeInsets.all(16),
+            child: Column(children: [
+              Row(crossAxisAlignment: CrossAxisAlignment.start, children: [
+                Text(
+                  performedAction.action.name,
+                  style: Theme.of(context).textTheme.bodyLarge,
+                ),
+                const Spacer(),
+                ElevatedButton(
+                    onPressed: null,
+                    child: Text(AppLocalizations.of(context)!
+                        .patientScreenPerformedActionDetails)),
+                MediaInfo(
+                    title: performedAction.action.name,
+                    media: performedAction.action.media)
+              ]),
+              const SizedBox(height: 8),
+              Row(crossAxisAlignment: CrossAxisAlignment.start, children: [
+                const Icon(ManvIcons.time, size: 20),
+                const SizedBox(width: 4),
+                Text(
+                  AppLocalizations.of(context)!
+                      .patientScreenPerformedActionTime,
+                  style: Theme.of(context).textTheme.bodyMedium,
+                ),
+                Expanded(
+                    child: Text(
+                  textAlign: TextAlign.right,
+                  TimeService.formatDateTime(
+                      performedAction.startTime, context),
+                  style: Theme.of(context).textTheme.bodyMedium,
+                ))
+              ]),
+              Row(crossAxisAlignment: CrossAxisAlignment.start, children: [
+                const Icon(ManvIcons.action, size: 20),
+                const SizedBox(width: 4),
+                Text(
+                  AppLocalizations.of(context)!
+                      .patientScreenPerformedActionResults,
+                  style: Theme.of(context).textTheme.bodyMedium,
+                ),
+                Expanded(
+                    child: Text(
+                  textAlign: TextAlign.right,
+                  overflow: TextOverflow.ellipsis,
+                  performedAction.action.results.join(', '),
+                  style: Theme.of(context).textTheme.bodyMedium,
+                ))
+              ])
+            ])));
+  }
+
+  PerformedActions sortedPerformedActions(Patient patient) {
+    final performedActions = patient.performedActions;
+    if (sortOldestFirst) {
+      performedActions.sort((a, b) => a.startTime.compareTo(b.startTime));
+    } else {
+      performedActions.sort((a, b) => b.startTime.compareTo(a.startTime));
+    }
+    return performedActions;
```

Comment:
I think (didn't test) that this method actually sorts in place and returns the sorted list. (Which is fine with me as we don't depend on them being in the original order anywhere?)